### PR TITLE
MDBOOK set up mostly empty pages and chapter structure ideas

### PR DESCRIPTION
**Pull request by Connoropolous**
I like how in the rust book, they come up with more interesting chapter names, like, "5. Using Structs to Structure Related Data" which defines the section really clearly. I would like to work on developing the section names in that direction.

- add empty pages with relevant section titles
- move extra README content into book, and link to it from README
screen shot 2018-07-29 at 4 54 42 pm

### CODE REVIEW

**Comment by Connoropolous** [comment]
@thedavidmeister @zippy @lucksus @ddd-mtl would you be willing to give this a quick and early look-over? 
Pondering whether to submit my docs as many small pull requests or do one longer, and larger ongoing one

**Comment by Connoropolous** [comment]
@zippy I totally agree. I need to continue to deepen and broaden my understanding, to start crafting meaningful high levels labels and categories, and to conceptualize a framework to fit things in. These are like the raw building blocks

**Comment by thedavidmeister** [comment]
@Connoropolous merge conflicts, soz my bad!

if you fix the merge conflicts and review my docs in the process i'm happy for you to 🚢 

i agree with high level splitting too, i was expecting to see something roughly like (based on our friday f2f):

- planning a dApp (high level, relatively non technical, concepts, get people onboarded, could be used by a visionary/founder to take a project/business to the world)
- building on hc application (dApp dev deets, diagrams, workflows, hooks/api functions, composers, extensibility)
- extending the hc platform (protocol level deets, contributing to hc-rust, building an compatible implementation in another language, hc implementation)

that can be discussed in a followup though :D

**Comment by Connoropolous** [comment]
I'll fix those up @thedavidmeister no problem. And I like the structure suggestion. I am looking at focusing on the most technical aspects first, and building up to the types of content you mentioned in "planning a dApp"

File: docker/Dockerfile.mdbook [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM sphinxc0re/holochain-rust
+FROM holochain/holochain-rust:develop
```
Comment (Connoropolous) [inline docker/Dockerfile.mdbook]: @sphinxc0re actually, this is what fixes the mdbook docker command and it's already in this PR, unless you want to implement a different solution.  #178

Comment (sphinxc0re) [reply docker/Dockerfile.mdbook]: No, that's fine. That's exactly how I would've fixed it

**Pull request merged**